integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.6.23 Version bump reason: Updating ZAP to v2022.12.17-nightly
+0.6.24 Version bump reason: [Telink] Update Telink Docker image (Zephyr update)
```

Comment:
Note that you will need a different version here....  0.6.24 already exists.

Reply:
Fixed, and maybe will fix again till this PR will be merged)